helm-charts/codegen-openshift-rhoai/react-ui/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Default values for codegen-react-ui.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  clusterDomain: "insert-your-cluster-domain-here"
+
+image:
+  repository: "insert-your-image-here"  # image-registry.openshift-image-registry.svc:5000/<namespace>/react-ui
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+source:
+  contextDir: CodeGen/ui
+  gitRef: main
```

Comment:
Shouldn't we reference specific release version instead of main branch?

Reply:
Changed _main_ to specific release